even-more-fish-plugin/src/main/java/com/oheers/fish/competition/Competition.java

```diff
@@ -183,6 +183,10 @@ private Message getTypeFormat(ConfigMessage configMessage) {
             else message.setFishCaught(selectedFish.getName());
         } else if (competitionType == CompetitionType.SPECIFIC_RARITY) {
             message.setAmount(Integer.toString(numberNeeded));
+            if (selectedRarity == null) {
+                EvenMoreFish.getInstance().getLogger().log(Level.WARNING, "Null rarity found. Please check your config files.");
```

Comment:
You can use a convinience function here. `EvenMoreFish.getInstance().getLogger().warning("Null rarity found. Please check your config files.")`. Instead of specifying the level.